addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/discovery/FreeboxServerDiscoveryParticipant.java

```diff
@@ -71,12 +72,15 @@ public DiscoveryResult createResult(ServiceInfo service) {
         if (thingUID != null && ip != null) {
             logger.info("Created a DiscoveryResult for Freebox Server {} on IP {}", thingUID, ip);
             Map<String, Object> properties = new HashMap<>(1);
-            properties.put(FreeboxServerConfiguration.IP_ADDRESS, ip);
+            properties.put(FreeboxServerConfiguration.IP_ADDRESS, ip + ":" + service.getPort());
```

Comment:
Yes, I think the property name is then misleading, it would propably be better to name it FQDN